udata_gouvfr/theme/gouvfr/templates/home.html

```diff
@@ -51,6 +51,7 @@ <h2 class="my-md text-white fs-hg">{{ post.name }}</h2>
             {# End last reuses #}
 
             {# Last inventories #}
+            {# this isn't displayed on the new version.
```

Comment:
It's because all the following code should be commented out, as it "isn't displayed on the new version". The closing comment tag is actually 20 lines later.
I guess we can soon plan to remove all the useless commented out parts and `_old` files !